src/modules/statistics/GoalSection.tsx

```diff
@@ -39,7 +39,7 @@ function KudoMeter(props: KudoMeterProps) {
         }}
       >
         <Icon
-          name={"lock"}
+          name={props.goal.achievedOn ? "lock_open" : "lock"}
```

Comment:
In the test the functionality had an open lock for goals achieved (makes sense) so added that back